Makefile

```diff
@@ -0,0 +1,9 @@
+kubeconfig := $(KUBECONFIG)
+## Following kubeconfig path is only valid from CI
+ifeq ($(RUN_FROM_CI),"true")
+	kubeconfig := "${HOME}/assets/auth/kubeconfig"
+endif
+
+.PHONY: run-e2e-tests
+run-e2e-tests:
```

Comment:
Can we have a kube-hunter target separately? This way we can only repeat one step if it fails, instead of running all e2e tests (I assume we will have more in the future.

Reply:
I wanted to reduce the code on CI repo and have a single point of entry for it. I can totally make changes to have additional target for kube-hunter and that be called from this one. Does that sound good?

Reply:
Yeah, that's what I meant. I suggest having `make kube-hunter` and `make run-e2e-tests`, which will be triggering `kube-hunter` target.

Reply:
done.